aya/src/maps/stack_trace.rs

```diff
@@ -159,6 +159,21 @@ impl<'a, T: Borrow<MapData>> IntoIterator for &'a StackTraceMap<T> {
     }
 }
 
+/// A resolver for symbols based on an address obtained from a stacktrace.
+pub trait SymbolResolver {
+    /// Resolve a symbol for a given address, if possible.
+    fn resolve_sym(&self, addr: u64) -> Option<String>;
```

Comment:
Should this return `Option<&str>` or perhaps `Option<Cow<'_, str>>`?

Reply:
Yeah probably Cow

Reply:
Didn't get notified for this, whoops. Should now be fixed.